package.json

```diff
@@ -376,6 +376,111 @@
+                      "description": "Number of attempts to generate a proof fix for each proof in one round. Warning: increasing `proofFixChoices` can lead to exponential growth in generation requests if `maxRoundsNumber` is relatively large.",
+                      "default": 1
+                    },
+                    "proofFixPrompt": {
+                      "type": "string",
+                      "description": "Prompt for the proof-fix request that will be sent as a user chat message in response to an incorrect proof. It may include the `${diagnostic}` substring, which will be replaced by the actual compiler diagnostic.",
+                      "default": "Unfortunately, the last proof is not correct. Here is the compiler's feedback: `${diagnostic}`. Please, fix the proof."
+                    },
+                    "maxPreviousProofVersionsNumber": {
+                      "type": "number",
+                      "description": "Maximum number of previous proof versions to include in the proof-fix chat, each presented as a dialogue: the user's diagnostic followed by the assistant's corresponding proof attempt. The most recent proof version being fixed is always included and is not affected by this parameter.",
+                      "default": 100
+                    }
+                  },
+                  "default": {
+                    "maxRoundsNumber": 1,
+                    "proofFixChoices": 1,
+                    "proofFixPrompt": "Unfortunately, the last proof is not correct. Here is the compiler's feedback: `${diagnostic}`. Please, fix the proof.",
+                    "maxPreviousProofVersionsNumber": 100
+                  }
+                }
+              }
+            },
+            "default": [
+              {
+                "modelId": "deep-seek-v3",
+                "modelName": "deepseek-chat",
+                "temperature": 1,
+                "apiKey": "None",
+                "choices": 15,
+                "systemPrompt": "Generate proof of the theorem from user input in Coq. You should only generate proofs in Coq. Never add special comments to the proof. Your answer should be a valid Coq proof. It should start with 'Proof.' and end with 'Qed.'.",
+                "maxTokensToGenerate": 2048,
+                "tokensLimit": 4096,
+                "maxContextTheoremsNumber": 100,
+                "multiroundProfile": {
+                  "maxRoundsNumber": 1,
+                  "proofFixChoices": 1,
+                  "proofFixPrompt": "Unfortunately, the last proof is not correct. Here is the compiler's feedback: '${diagnostic}'. Please, fix the proof.",
+                  "maxPreviousProofVersionsNumber": 100
+                }
+              }
+            ],
+            "markdownDescription": "List of configurations for DeepSeek models. Each configuration will be fetched for completions independently in the order they are listed.",
+            "order": 4
```

Comment:
I agree that the latest service we add could be placed in the 4th place, but if you decide that, then try to stay consistent. That means that in the all code further, it'd be better to put "DeepSeek" in the same place always, for example, after "LMStudio" kinda, just for readability.

Reply:
Yeap, I guess I placed the DeepSeek service everywhere on the last position. If I accidently swapped it somewhere, I will fix that.